docs/src/main/sphinx/release/release-406.md

```diff
@@ -0,0 +1,96 @@
+# Release 406 (18 Jan 2023)
+
+## General
+
+* Add support for [exchange spooling on HDFS](fte-exchange-hdfs) when
+  fault-tolerant execution is enabled. ({issue}`15160`)
+* Improve planner estimates for queries containing outer joins over a subquery
+  involving `ORDER BY` and `LIMIT`. ({issue}`15428`)
+* Improve accuracy memory usage reporting for table scans. ({issue}`15711`)
```

Comment:
```suggestion
* Improve accuracy of memory usage reporting for table scans. ({issue}`15711`)
```

Also in general maybe we should add a note regarding all memory usage reporting improvements that it's normal to see "more memory" being used in UI (but same usage from JVM heap) when upgrading Trino?